### PR TITLE
Select Popover Component

### PR DESCRIPTION
**Pull request by JonTheBurley**
This PR adds a Select Popover component. I also removed an import from the docs package that might break when upgrading as well as changed the baseUrl in the tsconfig so stories would find paths properly.

image
(Button not included in component. It wraps any children and shows under them.)

### CODE REVIEW

File: src/SelectPopover/SelectPopover.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+        if (renderInPortal && bodyElement) {
+            const portalElement = document.createElement('div');
+
+            if (!portal) {
+                bodyElement.appendChild(portalElement);
+            }
+
+            if (popoverElement) {
+                if (portal) {
```
Comment (zach-jones-slalom) [inline src/SelectPopover/SelectPopover.tsx:55]: Could this be simplified to a single if with if (popoverElement && portal)?

**Review by zach-jones-slalom — APPROVED** [review]

File: src/SelectPopover/SelectPopover.tsx (hunk 1 above)
Comment (JonTheBurley) [reply src/SelectPopover/SelectPopover.tsx]: Done.

**Review by jason-ransom-slalom — APPROVED** [review]
Pretty intense component. Nice work 👍🏾

File: src/Button/iconButtonVariants.ts [hunk 2]
```diff
@@ -50,7 +50,8 @@ export const getIconStyles = ({ alternate, disabled, theme }: GetIconStylesProps
                         color: theme?.colors?.white
                     }
                 }
-            })
+            }),
+            border: `1px solid ${theme?.colors?.saturatedRed}`
```
Comment (JonTheBurley) [inline src/Button/iconButtonVariants.ts:54]: The destructive icon-only button style was missing a border.

**Review by michaelpuzon — APPROVED** [review]

**Pull request merged**